pages/common/cowsay.md

```diff
@@ -7,7 +7,7 @@
 
```

Comment:
The commands LGTM. I would suggest adding a line about its depreciation above the more information link.

Reply:
Please write a suggestion what the deprecation notice has to be and I accept it.